bench/chromium/test_reqbench.py

```
@@ -7385,6 +7393,54 @@ def test_make_database_is_from_a_parsable_makefile(self):
+        # `make` invoked as a command in non-comment text: a flag or a target
+        # follows it. Prose about make in comments does not count.
+        make_call = re.compile(r"(?:^|[\s;&|(`])make\s+(?:-[A-Za-z]|[A-Za-z_][\w./-]*)(?=\s|$)")
+        offenders = []
+        for target in leased:
+            for line in self.recipes.get(target, []):
+                for rel in sorted(set(script_ref.findall(line))):
+                    path = os.path.join(self.REPO, rel)
+                    if not os.path.isfile(path):
+                        continue
+                    with open(path, encoding="utf-8") as f:
+                        code = "\n".join(l for l in f.read().splitlines()
```

Ruff (0.16.3)

[error] Ambiguous variable name: `l`

(E741)